config/owasp/suppressions.xml

```diff
@@ -1,12 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress until="2030-01-01">
+  <suppress>
     <notes><![CDATA[
-     Suppressing as it's a false positive (see: https://pivotal.io/security/cve-2018-1258)
-   ]]></notes>
-    <gav regex="true">^org\.springframework\.security:spring-security-crypto:5.3.[0-4].RELEASE</gav>
-    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    file name: spring-security*.jar
+        ]]></notes>
+    <packageUrl regex="true">^pkg:maven*.*$</packageUrl>
     <cve>CVE-2018-1258</cve>
   </suppress>
-
+  <suppress until="2021-01-05">
+    <notes><![CDATA[
+   file name: oauth2-oidc-sdk-7.1.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.nimbusds/oauth2\-oidc\-sdk@.*$</packageUrl>
+    <cve>CVE-2007-1651</cve>
+    <cve>CVE-2007-1652</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: tomcat-embed-core-9.0.38.jar
+   ]]></notes>
```

Comment:
please try find alternative as it is new cve.

Reply:
we have got approval for that